learning_journal/views/default.py

```diff
@@ -11,6 +11,13 @@
 @view_config(route_name='home', renderer='../templates/home.jinja2')
 def home_view(request):
     """Grab homepage data from db and send it to jinja."""
+    if request.method == "POST":
+        title = request.POST['title']
+        body = request.POST['body']
+        creation_date = datetime.strptime(request.POST['creation_date'], "%Y-%m-%d")
+        entry = Entry(title=title, body=body, creation_date=creation_date)
+        request.dbsession.add(entry)
+        return HTTPFound(location=request.route_url('home'))
```

Comment:
Why copy all your new entry code into home view?  You use an AJAX call to make the post, not a form action.